common/build/eslint-config-fluid/src/fluid-internal/README.md

```diff
@@ -0,0 +1,30 @@
+# eslint-plugin-fluid-internal
+
+This package contains custom ESLint rules specific for Fluid Framework. It is a separate local module which is exported as a plugin to the `@fluidframework/eslint-config-fluid` package (consumed in `minimal.js`'s `plugins` array).
+
+An ESLint Plugin is an extension for ESLint that adds additional custom rules and configuration options.
+
+See [ESLint: Name a Plugin](https://eslint.org/docs/latest/extend/plugins) for more details.
+
+## Background
+
+The `eslint-plugin-fluid-internal` is a nested package living inside the `@fluidframework/eslint-config-fluid` package. This structure is induced by the `ESLint`'s requirements which enforce:
```

Comment:
Nit: we should mention that we made this choice because of this requirement **and** we don't want to have to maintain 2 separate packages.